practice/src/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/main.css">
+  <title>Products Management</title>
+</head>
+
+<body>
+  <!-- Edit Feature -->
```

Comment:
@JennyDoJD I saw duplicate HTML code for the product form. Could you help double-check and find the best way to deal with it?